src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+}
+
+.star-block {
+  display: block;
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
```

Comment:
remove this justifyContent and set `margin-right: 4px` for each star exept the last one. It will allow you to pass tests I guess